e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts

```diff
@@ -128,6 +128,101 @@ test.describe.serial('Test RBAC plugin as an admin user', () => {
 
     await rolesHelper.deleteRole('role:default/test-role1');
   });
+
+  test('Create a role with a permission policy per resource type and verify that the only authorized users can access specific resources.', async () => {
```

Comment:
Sorry but it's not clear to me, does this test verify if the user can read/create/update/delete a catalog entity when any condition is satisfied? (assuming it's following [RHIDP-1941](https://issues.redhat.com/browse/RHIDP-1941))